### PR TITLE
change architecture according Andre

### PR DESCRIPTION
**Pull request by AKot1988**
Hi! Please check updated arch.
Update structure, add 'Rubic' roman font, ubdated variables synt.

Pay atetntiion for font mixins and font family in vatiables? becaus i'm not sure thats work correctly.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <title>RentCarService Vite + TS</title>
   </head>
   <body>
-    <div id="app"></div>
+    <div id="app">Hello dirty animal</div>
```
Comment (ohiienko-r) [inline index.html]: Why do we keep this in code?

**Review by ohiienko-r — CHANGES_REQUESTED** [review]
Please remvoe any text from HTML file.

File: index.html (hunk 1 above)
Comment (AKot1988) [reply index.html]: text has been cleaned

**Review by ohiienko-r — CHANGES_REQUESTED** [review]
Remove ANY strings within <div> element

**Review by ohiienko-r — APPROVED** [review]
LGTM

**Pull request merged**